Jenkinsfile

```diff
@@ -814,6 +835,7 @@ pipeline {
                     }
                     steps {
                         job_step_update(
+                            // ignbore_failure=true for test
```

Comment:
```suggestion
                            // ignore_failure=true for test
```

Reply:
That comment will be gone once this PR is ready for actual reviews.